core/include/detray/utils/invalid_values.hpp

```diff
@@ -39,7 +39,11 @@ template <typename T,
 DETRAY_HOST_DEVICE inline constexpr bool is_invalid_value(
     const T value) noexcept {
     if constexpr (std::is_signed_v<T>) {
-        return (math::abs(value) == detail::invalid_value<T>());
+        if constexpr (std::is_floating_point_v<T>) {
+            return (math::fabs(value) == detail::invalid_value<T>());
```

Comment:
Note that we arrive at this line with the traccc code. 🤔 But is that not a bug actually? Is this function meant to be called with floating point numbers?

Reply:
I don't know if it is actually called with floating point values anywhere, but it could be. To test e.g against ```std::numeric_limits<scalar_t>:max()```. This code is not used to catch large values that arise in calculations, but values that were set as explicitly invalid somewhere. Is this implementation too dangerous?

Reply:
If you say that calling it with a floating point type is reasonable, I'm absolutely happy. I was just not sure if an invalid value was properly defined for `float` and `double` even. 🤔

Fine, then let's go for this.